sci-physics/rivet/rivet-2.5.2.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit python-single-r1 autotools bash-completion-r1
+
+MYP=Rivet-${PV}
+
+DESCRIPTION="Toolkit for validation of Monte Carlo HEP event generators"
+HOMEPAGE="http://rivet.hepforge.org/"
+
+SRC_URI="http://www.hepforge.org/archive/${PN}/${MYP}.tar.bz2
+	doc? ( https://rivet.hepforge.org/trac/browser/doc/refs.bib )"
+LICENSE="GPL-2"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc python static-libs"
+
+RDEPEND="
+	dev-libs/boost:0=
+	sci-libs/gsl:0=
+	sci-physics/fastjet:0=[plugins]
+	sci-physics/hepmc:0=
+	>=sci-physics/yoda-1.5.0:0=[python]
+	python? ( ${PYTHON_DEPS} )"
+DEPEND="${RDEPEND}
+	doc? ( app-doc/doxygen[latex,dot] )
+	python? ( dev-python/cython[${PYTHON_USEDEP}] )"
+
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+S="${WORKDIR}/${MYP}"
+
+src_unpack() {
+	unpack "${MYP}.tar.bz2"
+	# refs.bib is missing in tarball (reported upstream)
+	if use doc ; then
+		cp "${DISTDIR}"/refs.bib "$S"/doc || die
+	fi
+}
+
+pkg_setup() {
+	use python && python-single-r1_pkg_setup
+}
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		--pdfdir="${EPREFIX}/usr/share/doc/${P}/"
+		$(use_enable python pyext) \
+		$(use_enable static-libs static) \
+		$(use_enable doc doxygen) \
+		$(use_enable doc pdfmanual)
+}
+
+src_compile() {
+	use doc && export VARTEXFONTS="${T}/fonts"
+	default
+	use doc && doxygen Doxyfile
+}
+
+src_install() {
+	use doc && HTML_DOCS+=( doxy/html/. )
+	default
+	newbashcomp "${ED%/}"/usr/share/Rivet/rivet-completion rivet
```

Comment:
`newbashcomp` is a kinda an install command, installing the file, which means you generally need to remove it afterwards from its original location